src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/DynamicFiles/DefaultDynamicDocumentContainer.cs

```diff
@@ -47,4 +47,10 @@ public IRazorDocumentPropertiesService GetDocumentPropertiesService()
         // opt out of those features.
         return null!;
     }
+
+    /// <summary>
+    /// IRazorMapping service is intended to map spans and edits from roslyn to razor files. This service is not supported in this context.
```

Comment:
What context is this? IIRC it's closed files, right?

Though I don't follow why we wouldn't want those handled, in case a rename affects them?

Reply:
Good catch, and an easy implementation to add. It will also mean I can add tests for the implementation of the interface rather than just the LSP endpoint.